src/Plugin/AddPathPlugin.php

```diff
@@ -47,7 +47,8 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
         $identifier = spl_object_hash((object) $first);
 
         if (!array_key_exists($identifier, $this->alteredRequests)) {
-            $request = $request->withUri($request->getUri()
+            $request = $request->withUri(
+                $request->getUri()
                 ->withPath($this->uri->getPath().$request->getUri()->getPath())
```

Comment:
not sure that this is more readable than the previous formatting. should we indent this some more? or extract the argument on this line to a variable so we can put `$request->getUri()->withPath($prefixedPath)` on one line?

Reply:
this change must have been suggested/automatic. 

I like your suggestion though.